src/main/resources/i18n/strings.properties

```diff
@@ -336,6 +336,15 @@ preferences.contribute.registeredFor=Supporter certificate registered for %s
 preferences.contribute.noCertificate=Support Cryptomator and receive a supporter certificate. It's like a license key but for awesome people using free software. ;-)
 preferences.contribute.getCertificate=Don't have one already? Learn how you can obtain it.
 preferences.contribute.promptText=Paste supporter certificate code here
+preferences.contribute.thankYou=Thank you for supporting Cryptomator’s open-source development!
+preferences.contribute.donate=Donate
+preferences.contribute.sponsors=Sponsors
```

Comment:
I would suggest to use the verb "Sponsor" here. This makes the intent clearer: Become a sponsor! (why should we link to our already existing sponsors?"